### PR TITLE
Migrate Liqo Images to Alpine

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR migrates all Liqo container images to Alpine as base image and refactors the requests and limits for Liqo Auth service.

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -67,18 +67,19 @@ jobs:
...
-        - peering-request-operator
+        - init-vkubelet
+        - liqonet
         - liqo-webhook
-        - crd-replicator
-        - auth-service
+        - peering-request-operator
+        - scheduling-node-operator
         - webhook-configuration
+        - virtual-kubelet
```
Comment (giorio94) [inline .github/workflows/integration.yml:82]: If the order is alphabetical, this should be swapped.

File: build/advertisement-broadcaster/Dockerfile [hunk 2]
```diff
@@ -6,6 +6,6 @@ WORKDIR /go/src/github.com/liqotech/liqo
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=$(go env GOARCH) go build ./cmd/advertisement-broadcaster/
 RUN cp advertisement-broadcaster /usr/bin/advertisement-broadcaster
 
-FROM scratch
+FROM alpine:3.12.4
```
Comment (giorio94) [inline build/advertisement-broadcaster/Dockerfile:9]: Why not 3.13?

File: scripts/virtual-kubelet/dep.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+case "$(uname -m)" in
+  x86_64|amd64)
+    export ARCH=amd64
+    ;;
+  arm*|aarch64)
+    export ARCH=arm64
+    ;;
+esac
+
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/${ARCH}/kubectl && chmod +x ./kubectl && cp kubectl /usr/bin/kubectl
```
Comment (giorio94) [inline scripts/virtual-kubelet/dep.sh:11]: 1.20?

**Review by giorio94 — APPROVED** [review]
/lgtm

**Review by alacuku — APPROVED** [review]

**Pull request merged**